### PR TITLE
Add colors to code formatting

### PR DESCRIPTION
**Pull request by pevik**

### CODE REVIEW

File: example.tex [hunk 1]
```diff
@@ -36,8 +63,11 @@
 This text will be in the second column
 and on a second thought this is a nice looking
 layout in some cases.
-\begin{lstlisting}[language=C, showstringspaces=false]
+
+\begin{lstlisting}[language=C, showstringspaces=false, caption=C example]
+// SPDX-License-Identifier: GPL-2.0
```
Comment (Werkov) [inline example.tex]: Why the license here?
If for comment highlighting, I'd suggest a comment in the function. I don't think a snippet like that is even copyrightable :-)

File: example.tex [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
+    breaklines=true,
+    captionpos=b,
+    keepspaces=true,
+    numbers=left,
+    numbersep=5pt,
+    showspaces=false,
+    showstringspaces=false,
+    showtabs=false,
+    tabsize=2
+}
```
Comment (Werkov) [inline example.tex:29]: The [package docs](https://mirrors.nic.cz/tex-archive/macros/latex/contrib/listings/listings.pdf) says:

> At the moment there are five such preferences files:
> 1. listings-acm.prf
> 2. listings-bash.prf
> 3. listings-fortran.prf
> 4. listings-lua.prf
> 5. listings-python.prf

Curiously, C is not among them.

> All contributors are invited to supply more personal preferences

Don't you want to contribute this to the listings package and only `\input` it in the example?

**Review by Werkov — CHANGES_REQUESTED** [review]
Thanks for visual improvement. I wonder if more higher upstream is possible :-)

File: example.tex (hunk 1 above)
Comment (pevik) [reply example.tex]: Yes, that was for comment, fixed.

File: example.tex (hunk 2 above)
Comment (pevik) [reply example.tex:29]: C specific listings would be great, but I wonder if/when I get time. At least they have git:

https://git.gnu.org.ua/listings.git/

And C would go to https://git.gnu.org.ua/listings.git/tree/lstdrvrs.dtx

Comment (Werkov) [reply example.tex:29]: Anyway, it's an example afterall.

**Comment by Werkov** [comment]
Thanks for the style!
Merged locally (with minor fixup + rebuild).

**Pull request closed without merging**